src/urdf_parser_py/display_urdf.py

```diff
@@ -1,20 +1,20 @@
-#!/usr/bin/env python
-
 import sys
 import argparse
 
 from urdf_parser_py.urdf import URDF
 
-parser = argparse.ArgumentParser(usage='Load an URDF file')
-parser.add_argument('file', type=argparse.FileType('r'),
-                    help='File to load. Use - for stdin')
-parser.add_argument('-o', '--output', type=argparse.FileType('w'),
-                    default=None, help='Dump file to XML')
-args = parser.parse_args()
 
-robot = URDF.from_xml_string(args.file.read())
+def main(argv=None):
```

Comment:
Nit: in ROS 2 we typically use `args` instead of `argv` (though we aren't 100% consistent, sigh).

Reply:
While `args/argv` argument has become widespread, I'm pretty sure it doesn't actually work. I think scripts still have to get arguments from `sys.argv`.

The template for `console_scripts` doesn't pass anything - see the function call on the entry point is just`()`.
https://github.com/pypa/setuptools/blob/990237c64404876791dc5985e7cfd260852dac0e/setuptools/command/easy_install.py#L2069

It's also not mentioned in the docs.
https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point